pkg/config/config.go

```diff
@@ -48,6 +50,30 @@ func (c *Config) BasicAuth() (user, pass string, ok bool) {
 	return user, pass, ok
 }
 
+// TLSCertFiles returns certificate and key files and an error if
+// both files doesn't exist and have approperiate file permissions
+func (c *Config) TLSCertFiles() (cert, key string, err error) {
+	if c.TLSCertFile == "" && c.TLSKeyFile == "" {
+		return "", "", nil
+	}
+
+	certFile, err := os.Stat(c.TLSCertFile)
+	if err != nil {
+		return "", "", fmt.Errorf("Could not access TLSCertFile: %v", err)
+	}
+
+	keyFile, err := os.Stat(c.TLSKeyFile)
+	if err != nil {
+		return "", "", fmt.Errorf("Could not access TLSKeyFile: %v", err)
+	}
+
+	if keyFile.Mode()&077 != 0 {
```

Comment:
this will not work on windows, please add GOOS check and skip check for windows.